_drafts/CocoaPods-1.9.0-beta.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title:  "CocoaPods 1.9 Beta has arrived!"
+author: amorde
+categories: cocoapods releases
+---
+
+_CocoaPods 1.9_ adds support for .xcframeworks, configuration based dependencies for pod authors, and other ehancements and bug fixes!
+
+<!-- more -->
+
+This release includes several quality-of-life enchancements in addition to important bug fixes.
+
+## XCFramework Support
+
+With the release of Xcode 11, Apple introduced a new bundle format using the `.xcframework` file extension. This format allows multiple copies of a framework compiled for different architectures and platforms to be combined into a single structure.
+
+This release introduces support for Pod authors to ship vendored `.xcframework` bundles using the existing [vendored_framework](https://guides.cocoapods.org/syntax/podspec.html#vendored_frameworks) DSL. Example:
+
+```ruby
+Pod::Spec.new do |s|
+  s.name         = 'ToastLib'
+  s.version      = '1.0.0'
+
+  # ...rest of attributes here
+
+  s.vendored_frameworks = 'ButterLib.xcframework'
+
+end
+```
+
+For apps that include Pods which vendor dynamic frameworks, a new script phase with the name `[CP] Prepare Artifacts` will be added to the project to facilitate including the .xcframework into the app bundle. Be on the look out for a blog post with more details on how this was implemented.
+
+For details on how to create an .xcframework, check out Apple's WWDC talk [Binary Frameworks in Swift](https://developer.apple.com/videos/play/wwdc2019/416/) which introduced the format.
+
+## Configuration-based dependencies for Podspecs
+
+CocoaPods has long supported excluding dependencies from configurations in which they are not needed. For example, debug-only dependencies used during development can be included using the `:configurations` option on the `pod` Podfile DSL:
+
+```ruby
+
+target 'BananaApp' do
+  pod 'Toast', :configurations => ['Debug']
+end
+```
+
+This release extends this functionality to Pod authors. The same `:configurations` option may now be used with the `dependency` Podspec DSL:
+
+
+```ruby
+Pod::Spec.new do |s|
+  s.name         = 'ToastLib'
+  s.version      = '1.0.0'
+
+  # ...rest of attributes here
+
+  s.dependency 'ButterDebugging', :configurations => ['Debug']
+  s.dependency 'ErrorReportingTool', :configurations => ['Release']
+
```

Comment:
nit extra line

Reply:
I can't recall if its `configuration` (singular) or `configurations`...

Reply:
according to https://github.com/CocoaPods/Core/pull/586 it's plural, but yeah I had the same thought too.